src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -148,15 +145,12 @@ public void testBearerWrongPosition() throws Exception {
     public void testBasicAuthHeader() throws Exception {
         Settings settings = Settings.builder().put("signing_key", BaseEncoding.base64().encode(secretKeyBytes)).build();
         HTTPJwtAuthenticator jwtAuth = new HTTPJwtAuthenticator(settings, null);
-        JwtParser jwtParser = Mockito.spy(JwtParser.class);
```

Comment:
out of curiosity, why did we remove the parser?

Reply:
This test was using an old version of Mockito and the functionality was deprecated, however when I looked at how the test was written this wasn't used so I removed it.

Reply:
okay